packages/twenty-server/src/engine/workspace-manager/standard-objects-prefill-data/views/opportunity-table-by-stage.view.ts

```diff
@@ -0,0 +1,115 @@
+import { ObjectMetadataStandardIdToIdMap } from 'src/engine/metadata-modules/object-metadata/interfaces/object-metadata-standard-id-to-id-map';
+
+import { OPPORTUNITY_STANDARD_FIELD_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-field-ids';
+import { STANDARD_OBJECT_IDS } from 'src/engine/workspace-manager/workspace-sync-metadata/constants/standard-object-ids';
+
+export const opportunitiesTableByStageView = (
```

Comment:
should not be in the prefill?
Should be in the seeds but not in prefill

Reply:
It's actually on the same place as the other file dedicated to opportunities views

Reply:
but these opportunity views are useful for new workspace and this one is only for seed?

please create a brand new workspace and make sure this view is not present!